samples/openapi3/client/petstore/dart2/petstore_client_lib_fake/lib/api/fake_classname_tags123_api.dart

```diff
@@ -85,7 +85,7 @@ class FakeClassnameTags123Api {
     // FormatException when trying to decode an empty string.
     if (response.body != null && response.statusCode != HttpStatus.noContent) {
       return apiClient.deserialize(_decodeBodyBytes(response), 'ModelClient') as ModelClient;
-    }
-    return null;
+        }
+    return Future.value(null);
```

Comment:
Maybe try to fix the indent and return `Future<{{{returnType}}}>.value(null)` for analyzer strong mode.
This is just a nice to have if you feel like it.

Reply:
Good idea

Reply:
You forgot the semicolon :)